addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/SpiderJob.java

```diff
@@ -76,13 +77,25 @@ public boolean applyCustomParameter(String name, String value) {
                 url = value;
```

Comment:
If progress is null then we should just be verifying, so we shouldnt set the contextName or url. I dont think it will cause a problem but its an unexpected side effect. Its a bit picky but I've been bitten by these sort of things in the past, and usually well after they were implemented ;)